external-import/threatmatch/src/threatmatch.py

```diff
@@ -100,9 +100,10 @@ def _get_item(self, token, type, item_id):
         if r.status_code == 200:
             data = r.json()["objects"]
             for object in data:
-                object["description"] = BeautifulSoup(
-                    object["description"], "html.parser"
-                ).get_text()
+                if "description" in object:
+                    object["description"] = BeautifulSoup(
+                        object["description"], "html.parser"
+                    ).get_text()
```

Comment:
If object["description"] == None, this will break, it would be safer this way :
```suggestion
            for object in data:
                 object["description"] = BeautifulSoup(
                     object.get("description", ""), "html.parser"
                 ).get_text()
```